fhir-path/src/main/java/com/ibm/fhir/path/util/FHIRPathUtil.java

```diff
@@ -347,7 +363,7 @@ public static Temporal getTemporal(TemporalAccessor temporalAccessor) {
         }
         throw new IllegalArgumentException();
     }
-
+    
```

Comment:
I've configured my eclipse to match albert's and squash trailing whitespace.  its funny to see this PR going the other direction :-)

Reply:
I have the `expand trailing whitespace` option enabled :)

Reply:
added commit that removes trailing whitespace